src/models/emailService/index.ts

```diff
@@ -57,6 +57,10 @@ export default class EmailService {
       host: smtpTransportConfig.get('SMTP_HOST'),
       port: smtpTransportConfig.get('SMTP_PORT'),
       secure: smtpTransportConfig.get('SMTP_SECURE'), // true for 465, false for other ports
+      auth: {
+        user: smtpTransportConfig.get('SMTP_USER'),
+        pass: smtpTransportConfig.get('SMTP_PASS'),
+      },
```

Comment:
Surely this should be only if those credentials are present do we send the "auth" section?

Reply:
I think we can assume that auth is required and then add that as a feature if it's not anywhere